app/views/core_induction_programmes/modules/show.html.erb

```diff
@@ -4,7 +4,12 @@
     <% if policy(@course_module).edit? %>
       <%= govuk_link_to "Edit module content", edit_module_path(@course_module), button: true%>
     <% end %>
-    <h1 class="govuk-heading-l"> <%= @course_module.term_and_title %></h1>
+
+    <% if @cip_is_ambition %>
+      <h1 class="govuk-heading-l"> <%= "#{@course_module.term.capitalize} term: #{@course_module.title}" %></h1>
+    <% else %>
+      <h1 class="govuk-heading-l"> <%= @course_module.term_and_title %></h1>
+    <% end %>
```

Comment:
This has a potential of turning pretty horrific, when we eventually add a few more rules for a few more providers. Avoiding this complexity is why we should move this logic into its own human editable database field.

Reply:
V.good point!

Hopefully the changes should mean more provider specific changes aren't as terrible to do.